docker/docker.makefile

```diff
@@ -0,0 +1,33 @@
+.PHONY: help build-base build-interface build-interface-gpu
+
+help:
+		@echo "Makefile arguments:"
+		@echo ""
+		@echo "BASE_IMAGE - Ubuntu Version"
+		@echo "PYTHON_VERSION - python version"
+		@echo "TENSORFLOW_VERSION - tensorflow version"
+		@echo "PyTorch_VERSION - pytoch version"
+		@echo "JAX_VERSION - jax version"
+		@echo ""
+		@echo "Makefile commands:"
+		@echo "build-base"
+		@echo "build-qchem"
+		@echo "build-interface"
+		@echo "build-interface-gpu"
+
+build-base:
+    @docker build -t pennylane/base -f docker/pennylane.dockerfile .
```

Comment:
Something is wrong here as the build is failing on my machine at this line. Also, please ensure the indentation level matches throughout :slightly_smiling_face: